src/Catalyst.Core.Lib.Tests/UnitTests/Validators/TransactionValidatorTests.cs

```diff
@@ -139,26 +135,40 @@ public void TransactionValidator_ValidateTransactionSignature_returns_false_for_
                     SignatureType = SignatureType.TransactionPublic
                 }
             };
-            
-            subbedContext.GetSignatureFromBytes(Arg.Is(txSig.ToByteArray()), Arg.Is(privateKey.GetPublicKey().Bytes))
-               .ReturnsForAnyArgs(Substitute.For<ISignature>());
-            
-            subbedContext.Verify(Arg.Any<ISignature>(), // @TODO be more specific
-                    Arg.Is(validTransactionBroadcast.ToByteArray()),
-                    Arg.Is(txSig.SigningContext.ToByteArray())
-                )
-               .Returns(false);
-            
-            subbedContext.Sign(Arg.Is(privateKey), validTransactionBroadcast.ToByteArray(), txSig.SigningContext.ToByteArray())
-               .SignatureBytes.Returns(new byte[64]);
-            
+
             var transactionValidator = new TransactionValidator(subbedLogger, subbedContext);
 
             validTransactionBroadcast.Signature = txSig;
 
             var result = transactionValidator.ValidateTransaction(validTransactionBroadcast);
             result.Should().BeFalse();
         }
+
+        sealed class FakeContext : ICryptoContext
+        {
+            readonly ISignature _signature;
+            readonly bool _verifyResult;
+
+            public FakeContext(ISignature signature, bool verifyResult)
+            {
+                _signature = signature;
+                _verifyResult = verifyResult;
+            }
+
+            public int PrivateKeyLength { get; }
+            public int PublicKeyLength { get; }
+            public int SignatureLength { get; }
+            public int SignatureContextMaxLength { get; }
+            public IPrivateKey GeneratePrivateKey() { throw new NotImplementedException(); }
+            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
+            public IPublicKey GetPublicKeyFromBytes(byte[] publicKeyBytes) { throw new NotImplementedException(); }
+            public IPrivateKey GetPrivateKeyFromBytes(byte[] privateKeyBytes) { throw new NotImplementedException(); }
+            public byte[] ExportPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
+            public byte[] ExportPublicKey(IPublicKey publicKey) { throw new NotImplementedException(); }
+            
+            public ISignature Sign(IPrivateKey privateKey, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context) => _signature;
+            public ISignature GetSignatureFromBytes(byte[] signatureBytes, byte[] publicKeyBytes) => Substitute.For<ISignature>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetSignatureFromBytes'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4569935)